modules/webbrowser/tests/regression/web-property-sync/web-property-sync.inv

```diff
@@ -517,69 +606,24 @@
                 </MetaDataMap>
             </Processor>
-            <Processor type="org.inviwo.LineRenderer" identifier="Line Renderer" displayName="Line Renderer">
-                <PortGroups>
-                    <PortGroup content="default" key="geometry" />
-                    <PortGroup content="default" key="image" />
-                    <PortGroup content="default" key="imageInport" />
-                </PortGroups>
-                <InPorts>
-                    <InPort type="org.inviwo.Mesh.flat.multi.inport" identifier="geometry" id="ref0" />
-                    <InPort type="org.inviwo.Image.inport" identifier="imageInport" id="ref1" />
-                </InPorts>
-                <OutPorts>
-                    <OutPort type="org.inviwo.Image.outport" identifier="image" id="ref2" />
-                </OutPorts>
+            <Processor type="org.inviwo.LineRenderer" identifier="LineRenderer" displayName="Line Renderer">
+                <PortGroups />
                 <Properties>
                     <Property type="org.inviwo.LineSettingsProperty" identifier="lineSettings">
                         <Properties>
                             <Property type="org.inviwo.FloatProperty" identifier="lineWidth">
                                 <MetaDataMap>
                                     <MetaDataItem type="org.inviwo.StringStringStdUnorderedMapMetaData" key="SavedState" />
                                 </MetaDataMap>
-                                <value content="1.5" />
-                            </Property>
-                            <Property type="org.inviwo.FloatProperty" identifier="antialiasing" />
-                            <Property type="org.inviwo.FloatProperty" identifier="miterLimit" />
-                            <Property type="org.inviwo.BoolProperty" identifier="roundCaps" />
-                            <Property type="org.inviwo.BoolProperty" identifier="pseudoLighting" />
-                            <Property type="org.inviwo.BoolProperty" identifier="roundDepthProfile" />
-                            <Property type="org.inviwo.StipplingProperty" identifier="stippling">
-                                <Properties>
-                                    <Property type="org.inviwo.OptionPropertyEnumInt" identifier="stippleMode" />
-                                    <Property type="org.inviwo.FloatProperty" identifier="stippleLen" />
-                                    <Property type="org.inviwo.FloatProperty" identifier="stippleSpacing" />
-                                    <Property type="org.inviwo.FloatProperty" identifier="stippleOffset" />
-                                    <Property type="org.inviwo.FloatProperty" identifier="stippleWorldScale">
-                                        <visible content="0" />
-                                    </Property>
-                                </Properties>
+                                <value content="2" />
```

Comment:
I think that this is what causes regression to fail. The script expects the value to be != 2 at startup. 
```
# Line width will be set to two when the webpage has loaded
while (lineWidth.value != 2): 
    inviwopy.qt.update();
```

Reply:
How would this apply to the other test with a static page? There is no dependency on any property.

Reply:
I tried to get it to work with the isLoading_ property, but it was too complicated (it goes from not loading | loading | not loading, e.g. false | true | false)

Reply:
That solved it for the property sync test, but not the other one.

Reply:
`CefLoadHandler` has a `OnLoadEnd` function when done loading the frame. Could we use this mechanism?

https://magpcss.org/ceforum/apidocs3/projects/(default)/CefLoadHandler.html#OnLoadEnd(CefRefPtr<CefBrowser>,CefRefPtr<CefFrame>,int)